src/rust/engine/fs/fs_util/src/main.rs

```diff
@@ -98,10 +105,13 @@ fn execute(top_match: clap::ArgMatches) -> Result<(), String> {
               let (fingerprint, size_bytes) = save_file(store, &posix_fs, f).wait().unwrap();
               Ok(println!("{} {}", fingerprint, size_bytes))
             }
-            o => Err(format!(
-              "Tried to save file {:?} but it was not a file, was a {:?}",
-              path,
-              o
+            o => Err((
+              format!(
+                "Tried to save file {:?} but it was not a file, was a {:?}",
+                path,
+                o
+              ),
+              1,
```

Comment:
Would be good to have some `static`/`const` values to give the various exit codes names.

Reply:
...or better yet: an `enum`.